gluster-exporter/main.go

```diff
@@ -116,6 +117,9 @@ func main() {
 	}
 	gluster = glusterutils.MakeGluster(&glusterConfig)
 
+	// Set the cluster ID
+	clusterID = glusterutils.GetClusterID()
```

Comment:
If this cluster ID set while running exporter then this will not picked up

Reply:
So we should call the function every time we need cluster id?

Reply:
thinking.. else we need to restart exporter to pick this.

Reply:
Also did you verified that this env variable is available when started using `systemctl start gluster-exporter`. Because I remember some settings done while adding Gd2 ReST Endpoints env variable support https://github.com/gluster/glusterd2/blob/master/extras/nightly-container/provision.yml#L129  (PR: https://github.com/gluster/glusterd2/pull/1308)

Reply:
So we need to add an entry here https://github.com/gluster/glusterd2/blob/master/extras/nightly-container/provision.yml#L129 ? `value: GLUSTER_CLUSTER_ID` ?